package.json

```diff
@@ -15,23 +15,35 @@
   "license": "MIT",
   "author": "Isaac J. Lee",
   "type": "module",
-  "main": "src/index.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "typesVersions": {
+    "*": {
+      "*": [
+        "dist/*"
+      ]
+    }
+  },
   "bin": "./bin/ember-codemod-pod-to-octane.js",
   "directories": {
     "test": "tests"
   },
   "files": [
-    "bin/",
-    "src/"
+    "dist"
```

Comment:
At the moment, I'm unsure whether an end-developer will be able to continue to use `npx` to run the codemod. Will check this after publishing a new version.